test/kldivergences.jl

```diff
@@ -0,0 +1,71 @@
+@testset "KL divergences" begin
+    function test_kl(p, q)
+        @test kldivergence(p, q) > 0
+        @test kldivergence(p, p) ≈ 0 atol=1e-1
+        @test kldivergence(q, q) ≈ 0 atol=1e-1
+        @test kldivergence(p, q) ≈ invoke(kldivergence, Tuple{MultivariateDistribution,MultivariateDistribution}, p, q; nsamples=10000) atol=1e-1
```

Comment:
Somehow I never got to use invoke,
I am trying
`invoke(kldivergence, Tuple{MultivariateDistribution,MultivariateDistribution}, p, q; nsamples=10000)` but I get an error `invoke: argument type error`

Reply:
You have to use `UnivariateDistribution` for univariate distributions, maybe this is the problem?

Reply:
But the whole point is to use the `MultivariateDistribution` one right? Anyway since I externalized the KL computation in an external function, we don't have to use this now

Reply:
Oh I see what you mean now